account_commission/readme/CONFIGURE.rst

```diff
@@ -3,4 +3,4 @@ For selecting invoice status in commissions:
 #. Edit or create a new record to select the invoice status for settling the commissions.
 
    * **Invoice Based**: Commissions are settled when the invoice is issued.
-   * **Payment Based**: Commissions are settled when the invoice is paid.
+   * **Payment Based**: Commissions are settled when the invoice is paid (or refunded).
```

Comment:
I think this requires a bit more of explanation:

```suggestion
   * **Payment Based**: Commissions are settled when the invoice is paid or refunded. Note that when refunding an invoice, the corresponding reversed commission will be settled as well, resulting in a 0 net commission between both operations.
```

Reply:
Done! The PR is ready.

Thanks for the review